userland/examples/nrf51_rng/Makefile

```diff
@@ -0,0 +1,11 @@
+# Makefile for user application
```

Comment:
You shouldn't need this app, you should be able to just use `userland/examples/tests/rng`. If that doesn't work, let's fix, but I'm guessing you just didn't see it because it's not documented anywhere :(